### DISCUSSION TITLE
请问 Mac 版鼠须管是否有参数可执行「同步用户数据」命令？

### DISCUSSION THREAD

**Discussion by someok** [rime/home#887] [Q&A]
想写个脚本定时执行「同步」命令，结果查了下源码好像没有提供？

![image](https://user-images.githubusercontent.com/1523082/134777283-affcb9a6-c333-4e14-a791-0eeb98257b5b.png)

**Comment by lotem** [comment]
同問： https://github.com/rime/squirrel/issues/492#issuecomment-770812620
